src/main/java/roomescape/repository/ThemeRepository.java

```diff
@@ -0,0 +1,78 @@
+package roomescape.repository;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import jakarta.annotation.PostConstruct;
+import roomescape.domain.Theme;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class ThemeRepository {
+
+	private static final RowMapper<Theme> THEME_ROW_MAPPER;
+
+	private final JdbcTemplate jdbcTemplate;
+
+	private SimpleJdbcInsert jdbcInsert;
+
+	ThemeRepository(JdbcTemplate jdbcTemplate) {
+		this.jdbcTemplate = jdbcTemplate;
+	}
+
+	@PostConstruct
+	public void init() {
+		this.jdbcInsert = new SimpleJdbcInsert(this.jdbcTemplate).withTableName("theme").usingGeneratedKeyColumns("id");
+	}
+
+	public List<Theme> findAll() {
+		String sql = """
+					SELECT id, name, description, thumbnail FROM theme
+				""";
+		return this.jdbcTemplate.query(sql, THEME_ROW_MAPPER);
+	}
+
+	public Theme findById(long id) {
+		String sql = "SELECT t.id, t.name, t.description, t.thumbnail FROM theme t WHERE t.id = ?";
+		return this.jdbcTemplate.queryForObject(sql, THEME_ROW_MAPPER, id);
+	}
+
+	public boolean isExistId(long id) {
+		String sql = "SELECT COUNT(*) FROM theme WHERE id = ?";
+		int count = this.jdbcTemplate.queryForObject(sql, Integer.class, id);
+		return count > 0;
+	}
+
+	public Theme save(Theme theme) {
+		Map<String, Object> parameters = new HashMap<>();
+		parameters.put("name", theme.getName());
+		parameters.put("description", theme.getDescription());
+		parameters.put("thumbnail", theme.getThumbnail());
```

Comment:
Java 9 부터 Map 을 포함한 collection 에 새로운 api 가 추가되었는데요,
이를 활용하면 더욱 편리하게 원하는 정보를 만들어볼 수 있어요!

```suggestion
        Map<String, Object> parameters = Map.of(
            "name", theme.getName(),
            "description", theme.getDescription(),
            "thumbnail", theme.getThumbnail());
```